benjaminkott/bootstrap_package
# 1318 accordion header level

# Pull Request

## Related Issues

* Resolves #1318

## Prerequisites

* [ ] Changes have been tested on TYPO3 v12.5 LTS
## Description

Add field items subheader for content elements
* accordion
* timeline
* icon group

![image](https://github.com/benjaminkott/bootstrap_package/assets/1639894/dd02dbc7-a124-4792-ba39-9154d609e4cc)


## Steps to Validate

1. pull
2. update database (add 1 field to tt_content)
2.1. default value in DB should be 4 (according to the old markup)
3. test 3 content elements in backend
4. test content elements in frontend
4.1. check if backend settings influence the output
4.1. check if "old" content elements stay with the h4

needs deploy to staging in order to change the subitem header levels according to the ticket (see above)
